src/main/edit_delete/postProcess.ts

```diff
@@ -0,0 +1,66 @@
+import { Transcription } from 'sharedTypes';
+import Clip from './Clip';
+
+/**
+ * Removes deleted words from edited trancription
+ * @param wordList
+ */
+const removeDeleted = (wordList: Array<any>): void => {
+  let i = 0;
+  while (i < wordList.length) {
+    if (wordList[i].deleted) {
+      wordList.splice(i, 1);
+    } else {
+      i += 1;
+    }
+  }
+};
+
+/**
+ * Processes edited data to be ready for export
+ * @param wordList
+ * @returns
+ */
+const FILENAME = 'placeholder_filename.mp4';
+
+const postProcess = (jsonTranscript: Transcription): Array<Clip> => {
+  const wordList = jsonTranscript.words;
+
+  removeDeleted(wordList);
+  const res: Array<Clip> = [];
+  let currentStartTime = -1;
+  let currentDuration = 0;
+
+  for (let i = 0; i < wordList.length - 1; i += 1) {
```

Comment:
This can be simplified too, though the actual implementation might be a bit tricky so I'm happy if you want to leave this for now. Just in general, outside of some very specific circumstances (like if you're implementing KMP or something lol) you generally don't need to use explicit for loops like this, `forEach` or `map` are a bit cleaner and easier to read

Think of the steps you could break this down into - e.g. firstly adding dummy words at the start and end to represent the max boundaries, then pairing each word with its successor, then combining words that overlap each others' boundaries, or something like that.